instagram/src/components/Comments/CommentCotainer.js

```diff
@@ -0,0 +1,93 @@
+import React from "react";
+import "./Comments.css";
+import Comment from "./Comment";
+import CommentInput from "./CommentInput";
+
+class CommentContainer extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      comments: [],
+      commentText: "",
+      likes: 0
+    };
+  }
+
+  componentDidMount = () => {
+    this.setState({
+      comments: this.props.comments,
+      likes: this.props.likes
+    });
+  };
+
+  addNewComment = (e, index) => {
+    e.preventDefault();
+    const newComment = { text: this.state.commentText, username: "marcelo" };
```

Comment:
...then use `.getItem()`. That's it. 👌 
```suggestion
    const newComment = { text: this.state.commentText, username: localStorage.getItem("user") };
```